### PR TITLE
Microwave

### PR DESCRIPTION
**Pull request by dvir001**
## About the PR
Added the upstream Microwave and anti ID in microwave code.

## Why / Balance
Make it impossible to microwave cards just to be able to hack into NFSD / SR Office.

## How to test
Spawn microwave
Add card
MMMMM
Explode.

## Media
N/A

## Breaking changes
N/A

**Changelog**
N/A

### CODE REVIEW

**Comment by MagnusCrowe** [comment]
Syndicate microwave?

**Comment by dvir001** [comment]
> Syndicate microwave?

Yes, from upstream
Its coming with the near update either way lamo

File: Resources/Prototypes/Entities/Structures/Machines/microwave.yml [hunk 1]
```diff
@@ -105,3 +107,21 @@
...
     - Chef
+
+- type: entity
+  id: SyndicateMicrowave
+  parent: KitchenMicrowave
+  name: donk co. microwave
+  description: So advanced, it can cook donk-pockets in a mere 2.5 seconds!
+  components:
+  - type: Microwave
+    cookTimeMultiplier: 0.5
```
Comment (whatston3) [inline Resources/Prototypes/Entities/Structures/Machines/microwave.yml:118]: Not a complaint, but this doesn't seem to actually get applied on spawns, might be conflicting with the machine parts being used - should test if it works upstream.

**Review by whatston3 — APPROVED** [review]
Works alright.  There seems to be a bug with the cook time reduction, haven't looked into why, but I think it's the capacitors being used - resulting cook multiplier should maybe be the product of a base value (which the syndicate microwave prototype sets) and the machine components used.

Microwave destruction works on both.  I should add the microwave board to the mail table (and clean up the noSpawn values on all the paper).

Sad we can't have a matching red SMEG-ish fridge.  Looking forward to buying a Bocadillo and being double-griefed because somebody forgot their ID in a ship console.

**Pull request merged**